docs/editor/userdefinedsnippets.md

```diff
@@ -164,6 +164,17 @@ The inserted text is matched with the regular expression and the match or matche
 Every occurrence of a placeholder can define its own transformation independently using the value of the first placeholder.
 The format for Placeholder-Transforms is the same as for Variable-Transforms.
 
+### Transform Examples
+
+The examples are shown within double quotes, as they would appear inside a snippet body, to illustrate the need to double escape certain characters. Sample transformations and the resulting output for the filename `example-123.456-TEST.js`.
+
+Example | Output | Explanation
+-------------|--------|------------
+`"${TM_FILENAME/[\\.]/_/}"` | `example-123_456-TEST.js` | Replace the first `.` with `_`
+`"${TM_FILENAME/[\\.-]/_/g}"` | `example_123_456_TEST_js` | Replace each `.` or `-` with `_`
+`"${TM_FILENAME/(.*)/${1:/upcase}/}"` | `EXAMPE-123.456-TEST.JS` | Change to all upperacse
```

Comment:
Should be EXAMPLE-123.456-TEST.JS, an 'L' is missing.

Reply:
Dang, my first docs pull request and managed to typo it, thanks for catching that.